samples/bot-formatting-cards/csharp/README.md

```diff
@@ -0,0 +1,158 @@
+---
+page_type: sample
```

Comment:
**Make sure that you have verified below points in your PR before sharing the PR:** 

1.  Add build pipeline for C#/NodeJS samples. 
2. Add deployment pipeline for every new sample like for C#, if sample exist otherwise create it for NodeJS sample.
3. Deploy the sample manually, test it and add the deployed manifest in your sample folder. 
4. Make sure that you have added the deployed manifest link in your Readme file. 
5. Make sure to link your sample with main/root readme. 
6. Make sure that you have linked the proper Microsoft Documentation with your sample's readme file in "Further reading" section. 
7. Once sample is pushed, make sure to raise a doc bug on Microsoft documentation to link your sample on that page. 
8. Make sure that you have added sample.json with proper links of .gif and metadata. 
9. Make sure to run build pipeline every time you push the new changes in PR, including PR comments fixes and add the screenshot/link with your PR. 
10. Verify that your readme file is visible properly in preview section. 

**Code related guidelines:** 

1. Remove all extra spaces and lines from your code. 
2. Make sure that you have added proper summary for each and every method. 
3. Don't push credentials in your PR. 
4. Verify alignments and correct those. 
5. Don't put any hardcoded value and push those in PR. 
6. Remove unwanted commented code 
7. Use latest framework and tools for your code implementation 
8. Add file header for every file. 
9. Read all the configuration from .env or appSettings.json (Do not add placeholders in code file.) 
10. Use latest manifest version and add proper descriptions. 
11. Use proper placeholders in manifest and define those in Readme file.